File: app/src/main/java/dev/carloszuil/herojourney/ui/journey/ConcentrationFragment.java
package dev.carloszuil.herojourney.ui.journey;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import dev.carloszuil.herojourney.databinding.FragmentConcentracionBinding;

public class ConcentrationFragment extends Fragment {

    private FragmentConcentracionBinding binding;

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater,
                             @Nullable ViewGroup container,
                             @Nullable Bundle savedInstanceState) {
        binding = FragmentConcentracionBinding.inflate(inflater, container, false);
        return binding.getRoot();
    }

    @Override
    public void onDestroyView() {
        super.onDestroyView();
        binding = null;
    }
}

File: app/src/main/java/dev/carloszuil/herojourney/ui/journey/ConcentrationFragment.kt
package dev.carloszuil.herojourney.ui.concentracion

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import dev.carloszuil.herojourney.databinding.FragmentConcentracionBinding

class ConcentrationFragment : Fragment() {

    private var _binding: FragmentConcentracionBinding? = null
    private val binding get() = _binding!!

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {
        _binding = FragmentConcentracionBinding.inflate(inflater, container, false)
        return binding.root
    }

    override fun onDestroyView() {
        super.onDestroyView()
        _binding = null
    }
}
